### PR TITLE
 #158423707 Add drivers expanded ride view 

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

* Adds a drivers view of an expanded ride offer's details.

* A driver shuold be able to view all the details associated with that particular ride offer, with an option to accept or reject requests from passengers.

* A driver should also be able to close down the ride offer at will (E.g When he is at full capacity)

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-drivers-expanded-ride-#158423707.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `expanded_ride_driver.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-drivers-expanded-ride-#158423707`

* Run `cd Ride-My-Way/UI/`

* Open `expanded_ride_driver.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158423707](https://www.pivotaltracker.com/story/show/158423707)

#### Screenshots 
expridedriver

### CODE REVIEW

**Comment by ghost** [comment]
The code is okay and design is good

**Comment by xcixor** [comment]
Great idea to have the status as a dropdown and if I may suggest white text color could be better on the black backgrounds

**Comment by meshack-mbuvi** [comment]
Same observations as @xcixo has made. Also, what would be the effect if you reduce the width of the labels for ride details? Try and see whether it becomes more user friendly

File: UI/expanded_ride_driver.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      <th>Departure Point</th>
+      <td>Thika Rd Mall</td>
+    </tr>
+    <tr>
+      <th>Departure Time</th>
+      <td>07:00AM</td>
+    </tr>
+    <tr>
+      <th>Final Destination</th>
+      <td>petergriffin@email.com</td>
```
Comment (JoshuaOndieki) [inline UI/expanded_ride_driver.html]: I assume your final destination is not an email address

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: UI/static/main.css [hunk 2]
```diff
@@ -120,4 +149,9 @@ footer a {
     .container {
         width: 75%;        
     }
+}
+
+/* Handle wide tables */
```
Comment (JoshuaOndieki) [inline UI/static/main.css]: :sparkles: Great work with handling the tables! :+1: :sparkles:

**Review by JoshuaOndieki — APPROVED** [review]

File: UI/expanded_ride_driver.html (hunk 1 above)
Comment (artorious) [reply UI/expanded_ride_driver.html]: Thanks for that :+1:

**Comment by artorious** [comment]
Hi @xcixor and @meshack-mbuvi 
I've increased the font size on the select buttons and changed color to white

@meshack-mbuvi please add color on `user-friendly`

**Comment by meshack-mbuvi** [comment]
It looks good. Though you can consider reducing the amount of black color.You can have black color occupying atmost 30% of your design

**Comment by LWanjiru** [comment]
@artorious Consider improving the look and feel of this page. There is a lot going on in terms especially of the background colours. Is it possible to have a more consistent feel? 

Curious question: what is the inspiration behind where the `status` drop-down appears?

**Comment by artorious** [comment]
@meshack-mbuvi @rethes @xcixor @JoshuaOndieki @LWanjiru Thank you for your input and direction. I have toned down the `black-heavy` look of the page, I now realize that it was distracting from the content. I've also moved the `Ride status` selector to a more practical and suited position.

expridedriver

**Pull request merged**